<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransfersAccountInfoWidget</class>
 <widget class="QWidget" name="TransfersAccountInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">#TransferManagerAccountInfo
{
	background-color: #FAFAFA;
}

#bUpgrade
{
	font-family: Inter;
	font-size: 12px;
	line-height: 16px;
	border: 1px solid #0000001A;
	border-radius: 4px;
	color: #FFFFFF;
	padding: 0px 8px;
	background-color:  #00BFA5;
}

#lStorage
{
	font-family: Inter;
	font-size: 12px;
	line-height: 18px;
	color: #303233;
}

#bUpgrade::hover
{
	background-color:  #00A787;
}

#bUpgrade::pressed
{
	background-color: #00D7C2;
}

#bUpgrade::disabled
{
	background-color: #00BFA54D;
}

#pbStorage
{
	border-style: none;
	border-radius: 4px;
	background-color: #EDEDED; /*colorToken.indicator-background*/
	max-height: 8px;
}

#pbStorage::chunk[state=&quot;default&quot;]
{
	background-color: #09BF5B; /*colorToken.indicator-green*/
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
}

#pbStorage::chunk[state=&quot;almost&quot;]
{
	background-color: #FB6514; /*colorToken.indicator-orange*/
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	border-top-right-radius: 0px;
	border-bottom-right-radius: 0px;
}

#pbStorage::chunk[state=&quot;almostfull&quot;]
{
	background-color: #FB6514; /*colorToken.indicator-orange*/
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
}

#pbStorage::chunk[state=&quot;full&quot;]
{
	background-color: #F63D6B; /*colorToken.indicator-pink*/
	border-top-left-radius: 4px;
	border-bottom-left-radius: 4px;
	border-top-right-radius: 4px;
	border-bottom-right-radius: 4px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>13</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="vlContainer">
     <property name="spacing">
      <number>13</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QWidget" name="wTop" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="hlTop">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="AccountTypeWidget" name="wAccountType" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bUpgrade">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wBottom" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>34</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="vlStorage">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QProgressBar" name="pbStorage">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>8</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>8</height>
           </size>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lStorage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>18</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountTypeWidget</class>
   <extends>QWidget</extends>
   <header location="global">AccountTypeWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
